Makefile.am

```diff
@@ -60,6 +60,27 @@ AM_CPPFLAGS = \
 # Utility library
 noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
 
+enum_header_files = \
+	libeos-updater-util/flatpak.h \
+	eos-updater-flatpak-installer/installer.h \
```

Comment:
Similarly here. It’s also a bit odd for libeos-updater-util’s `enums.[ch]` to contain type definitions for an enum which is declared in a different program (`eos-updater-flatpak-installer` rather than `libeos-updater-util`). Could the enums from `installer.h` be moved into `libeos-updater-util/types.h`, for example? Then the enum and its GType are defined in the same library.

Reply:
You can then list `libeos-updater-util/types.h` in `enum_header_files` above in this commit, to avoid leaving the list empty when it’s first introduced.

Reply:
Its probably easier to just move the enum generation into the commit that introduced the flatpak stuff in fetch.